locale/ta/LC_MESSAGES/django.po

```diff
@@ -2817,7 +2817,7 @@ msgstr ""
 #: templates/zerver/social_auth_select_email.html:7
 msgid ""
 "\n"
-"        <h1>Select email</h1>\n"
```

Comment:
Can you discard this change? These changes aren't necessary as @timabbott is responsible for pulling data from Transifex and updating the files in `locale` periodically.

Reply:
Yes, we should drop this commit.